vnext/ReactWindowsCore/WinRTWebSocketResource.cpp

```diff
@@ -0,0 +1,353 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// clang-format off
+#include "WinRTWebSocketResource.h"
+
+#include <winrt/Windows.Security.Cryptography.h>
+#include <Unicode.h>
+#include <Utilities.h>
+
+// Standard Library
+#include <future>
+
+using Microsoft::Common::Unicode::Utf8ToUtf16;
+using Microsoft::Common::Unicode::Utf16ToUtf8;
+using Microsoft::Common::Utilities::CheckedReinterpretCast;
+
+using std::function;
+using std::size_t;
+using std::string;
+using std::vector;
+
+using winrt::fire_and_forget;
+using winrt::hresult;
+using winrt::hresult_error;
+using winrt::resume_background;
+using winrt::Windows::Foundation::IAsyncAction;
+using winrt::Windows::Foundation::Uri;
+using winrt::Windows::Networking::Sockets::IWebSocket;
+using winrt::Windows::Networking::Sockets::MessageWebSocket;
+using winrt::Windows::Networking::Sockets::MessageWebSocketMessageReceivedEventArgs;
+using winrt::Windows::Networking::Sockets::SocketMessageType;
+using winrt::Windows::Networking::Sockets::WebSocketClosedEventArgs;
+using winrt::Windows::Security::Cryptography::CryptographicBuffer;
+using winrt::Windows::Storage::Streams::DataReader;
+using winrt::Windows::Storage::Streams::DataWriter;
+using winrt::Windows::Storage::Streams::UnicodeEncoding;
```

Comment:
this is a lot of boilerplate - just do using namespace instead, there's no need to name individual types